src/test/java/com/lambdaschool/foundation/controllers/UserControllerTest.java

```diff
@@ -0,0 +1,303 @@
+package com.lambdaschool.foundation.controllers;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.lambdaschool.foundation.models.City;
+import com.lambdaschool.foundation.models.User;
+import com.lambdaschool.foundation.repository.UserRepository;
+import com.lambdaschool.foundation.services.CityService;
+import com.lambdaschool.foundation.services.HelperFunctions;
+import com.lambdaschool.foundation.services.UserService;
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.mockito.Mockito;
+import static org.mockito.ArgumentMatchers.any;
+import static org.mockito.ArgumentMatchers.anyString;
+import static org.springframework.test.web.servlet.result.MockMvcResultMatchers.status;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.autoconfigure.web.servlet.WebMvcTest;
+import org.springframework.boot.test.mock.mockito.MockBean;
+import org.springframework.http.MediaType;
+import org.springframework.security.oauth2.jwt.JwtDecoder;
+import org.springframework.test.web.servlet.MockMvc;
+import org.springframework.test.web.servlet.MvcResult;
+import org.springframework.test.web.servlet.RequestBuilder;
+import org.springframework.test.web.servlet.request.MockMvcRequestBuilders;
+import org.springframework.test.web.servlet.result.MockMvcResultHandlers;
+import org.springframework.web.context.WebApplicationContext;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+//@RunWith(SpringRunner.class)
+//@SpringBootTest(webEnvironment = SpringBootTest.WebEnvironment.RANDOM_PORT,
+//    classes = FoundationApplication.class)
+//@AutoConfigureMockMvc
+//@WithMockUser(username = "admin")
```

Comment:
As per Ryan commented out code doesn't belong in Main. Please remove any unneeded annotations.

Reply:
I deleted this and all other instances of commented out code I could find.